modules/dr-restoring-cluster-state.adoc

```diff
@@ -38,14 +36,14 @@ This procedure assumes that you copied the `snapshot_db_kuberesources_<datetimes
 [core@ip-10-0-143-125 ~]$ export INITIAL_CLUSTER="etcd-member-ip-10-0-143-125.ec2.internal=https://etcd-0.clustername.devcluster.openshift.com:2380,etcd-member-ip-10-0-35-108.ec2.internal=https://etcd-1.clustername.devcluster.openshift.com:2380,etcd-member-ip-10-0-10-16.ec2.internal=https://etcd-2.clustername.devcluster.openshift.com:2380"
 ----
 
-.. Repeat these steps on your other master hosts, each in a separate terminal. Be sure to use the same etcd backup file on each master host.
+.. Repeat these steps on your other master hosts, each in a separate terminal. Be sure to use the same etcd backup directory on each master host.
```

Comment:
How about "Be sure to use the backup directory containing the same set of backup files on each master host?" Not sure "if use the same etcd backup directory" is clear to the end user.

Reply:
Good suggestion, done.